deployment/modules/space/main.tf

```diff
@@ -5,36 +5,31 @@ data "archive_file" "strapi-image-zip" {
   excludes    = concat(["**/.git"], split("\n", file(".cfignore")))
 }
 
-resource "cloudfoundry_user_provided_service" "login-gov" {
+data "cloudfoundry_user_provided_service" "login-gov" {
   name  = "login-gov"
   space = data.cloudfoundry_space.space.id
-  credentials = {
-    certificate = lookup(var.cf_strapi_logingov_cert, var.cf_env)
-    public_key  = var.cf_strapi_logingov_key
-    issuer      = lookup(var.cf_strapi_logingov_issuer, var.cf_env)
-  }
 }
 
 resource "cloudfoundry_service_instance" "strapi-api-db" {
-  name         = "strapi-api-db-${var.cf_env}"
+  name         = "strapi-api-db"
   space        = data.cloudfoundry_space.space.id
   service_plan = data.cloudfoundry_service.rds.service_plans[var.cf_rds_strapi_db_service_plan]
 }
 
 resource "cloudfoundry_service_instance" "strapi-image-bucket" {
-  name         = "strapi-image-bucket-${var.cf_env}"
+  name         = "strapi-image-bucket"
```

Comment:
Global name might conflict

Reply:
I removed all the env references from names, because the CloudFoundry space that we deploy into should act like a namespace, and it seemed simpler to just put the env description in the space name.